src/lib/components/table/proposals/ProposalTextCell.tsx

```diff
@@ -1,11 +1,26 @@
 import { Flex, Text } from "@chakra-ui/react";
 import { useRef, useState } from "react";
 
+import { MobileLabel } from "../MobileLabel";
 import { useMobile } from "lib/app-provider";
 import { DotSeparator } from "lib/components/DotSeparator";
 import { Expedited } from "lib/components/Expedited";
 import type { ProposalType } from "lib/types";
 
+const RenderMsg = ({ types }: { types: string[] }) =>
```

Comment:
`types` on mobile and desktop are rendering differently. Please double check the original code